src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -14,24 +14,28 @@ public class CommandResult {
     /** Help information should be shown to the user. */
     private final boolean showHelp;
```

Comment:
Please note boolean names should typically start with "is" or "has", maybe in this case "will".

Reply:
These were the names originally so I didn't change them, and they have functions with is- prefixed (e.g isShowHelp()) to return their values, so not sure if changing the name is needed in this case.

Reply:
@mynameizzhafeez @seanfirefox @kennycjy any thoughts on this issue?

Reply:
It would be better to have the "is" inside.